share/jupyterhub/templates/login.html

```diff
@@ -10,6 +10,7 @@
 
 {% block login %}
 <div id="login-main" class="container">
+  <!--<h1 class="sr-only">Login page</h1>-->
```

Comment:
Let's not leave commented-out HTML in here

```suggestion
```

Reply:
Okay.